lib/fluent/plugin_helper/retry_state.rb

```diff
@@ -123,32 +125,27 @@ def step
             @current = :secondary
             @secondary_transition_steps = @steps
           end
+
           @next_time = calc_next_time
+
+          unless @has_reached_timeout
+            @has_reached_timeout = @next_time >= @timeout_at
+          else
+            @has_timeouted = @next_time >= @timeout_at
+          end
```

Comment:
I don't prefer `unless` clause with `else` clause, double negative makes hard to read:
```suggestion
          if @has_reached_timeout
            @has_timeouted = @next_time >= @timeout_at
          else
            @has_reached_timeout = @next_time >= @timeout_at
          end
```

Reply:
fixed on 62b7002cdd09b452308e2ef869131f907022d904